cmd/admin-policy-list.go

```diff
@@ -76,11 +77,15 @@ func mainAdminPolicyList(ctx *cli.Context) error {
 	fatalIf(probe.NewError(e).Trace(args...), "Cannot list policy")
 
 	if policyName := args.Get(1); policyName != "" {
-		printMsg(userPolicyMessage{
-			op:         "list",
-			Policy:     policyName,
-			PolicyJSON: policies[policyName],
-		})
+		if fmt.Sprintf("%s", policies[policyName]) != "" {
```

Comment:
`policies` is of type `map[string][]byte`.
So, we can simply check the length of the byte array if zero or not, IMHO:
```
    if len(policies[policyName]) != 0 {
```

Reply:
:thumbsup: Modified.